src/main/java/fi/metatavu/dcfb/server/persistence/model/CategoryMeta.java

```diff
@@ -0,0 +1,105 @@
+package fi.metatavu.dcfb.server.persistence.model;
+
+import java.util.UUID;
+
+import javax.persistence.Cacheable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'javax.persistence.EntityListeners'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=7&issues=AWQh_VKt_bEzQ-kUa2zU&open=AWQh_VKt_bEzQ-kUa2zU)

[ISSUE_KEY]: AWQh_VKt_bEzQ-kUa2zU